numpy/core/tests/test_umath.py

```diff
@@ -17,6 +18,21 @@
     _gen_alignment_data, assert_array_almost_equal_nulp, assert_warns
     )
 
+def get_glibc_version():
+    ver = '0.0'
+    try:
+        ver = os.confstr('CS_GNU_LIBC_VERSION').rsplit(' ')[1]
+    except Exception as inst:
+        print(inst)
+
+    return ver
```

Comment:
Maybe
```
def get_glibc_version():
    try:
        ver = os.confstr('CS_GNU_LIBC_VERSION').rsplit(' ')[1]
    except OSError:
        ver = '0.0'
    return ver
```

Reply:
Looks like `os.confstr()` method can cause 3 types of exceptions: 

1. `OSError` (specified name isn't supported by the OS)
2. `ValueError` (specified name is not a valid configuration variable) 
3. `AttributeError: module 'os' has no attribute 'confstr'` (like on Windows). 

Isn't it okay to just use a general Exception instead of specifying all 3?

Reply:
Hmm. (Checks) Looks like `Exception` is OK and won't catch `KeyboardInterrupt` exceptions, which is what we want to avoid doing. So current code is OK.

Reply:
What I would like to get rid of is the print statement.